bio2zarr/tskit.py

```diff
@@ -0,0 +1,241 @@
+import logging
+import pathlib
+
+import numpy as np
+import tskit
+
+from bio2zarr import constants, core, vcz
+
+logger = logging.getLogger(__name__)
+
+
+class TskitFormat(vcz.Source):
+    def __init__(
+        self,
+        ts_path,
+        individual_nodes,
```

Comment:
Best make this ``individuals_nodes`` for consistency

Reply:
Fixed in 727e47d